pkg/controller/manager/manager_controller_test.go

```diff
@@ -566,6 +560,12 @@ var _ = Describe("Manager controller tests", func() {
 		})
 
 		Context("image reconciliation", func() {
+			BeforeEach(func() {
+				Expect(c.Create(
```

Comment:
This can be a single line - no need to split it up.

Alternatively, we could keep this in the main `BeforeEach` and then any tests that need something different can just call `Update` in their `BeforeEach` to modify the installation.